gtsfm/scene_optimizer.py

```diff
@@ -231,6 +239,25 @@ def _filter_dict_keys(dict: Dict[Any, Any], ref_dict: Dict[Any,Any]) -> Dict[Any
         return output_graph[0]
 
 
+def align_estimated_gtsfm_data(ba_input: GtsfmData, ba_output: GtsfmData) -> Tuple[Delayed]:
```

Comment:
so the return type should be 
```python
def align_estimated_gtsfm_data(ba_input: GtsfmData, ba_output: GtsfmData) -> Tuple[GtsfmData, GtsfmData]
```
this is because the function call is wrapped with `dask.delayed()`. We only mark the return type as dask.Dalayed when the function call is not wrapped.